### PR TITLE
Exp/deprecate flowtype

### PR DESCRIPTION
**Pull request by ethan605**
## Background

The TypeScript compiler forbids the uses of types in JS files. Hence we have to hide them and hopefully can recycle when we convert them into TypeScript.

Remove all Flowtype-compatible typings & enable build/dev process to run successfully

Following PRs: #1259 

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [n/a] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [n/a] Have any new automated tests been implemented or the existing ones changed?
- [n/a] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Review by kopijunkie — APPROVED** [review]
LGTM. Looking forward to a TypeScript future! 🎉

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1258-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1258-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json [hunk 1]
```diff
@@ -38,12 +38,19 @@
     "identity",
     "verification"
   ],
-  "files": ["dist", "lib", "src"],
+  "files": [
+    "dist",
+    "lib",
+    "src"
+  ],
```
Comment (stefaniacardenas) [inline package.json:45]: In the previous PR, this array was changed to be inline, now it's back as it was. Have we changed the linter rule?

File: src/components/Camera/index.js [hunk 2]
```diff
@@ -16,7 +14,7 @@ import style from './style.scss'
 // support the precise resolution.
 const DEFAULT_CAMERA_HEIGHT_IN_PX = 720
 
-export type Props = {
+/* export type Props = {
```
Comment (stefaniacardenas) [inline src/components/Camera/index.js:17]: Why are the flow types here and in the other files commented out? Are you going to use them as reference for the typescript implementation?

File: package.json (hunk 1 above)
Comment (ethan605) [reply package.json]: Not sure why, maybe because of the TypeScript parser 🤔

File: src/components/Camera/index.js (hunk 2 above)
Comment (ethan605) [reply src/components/Camera/index.js:17]: Sorry I forgot to explain in the description: the TypeScript transpiler forbids the uses of types in JS files. Hence we have to _hide_ them and hopefully can recycle when we convert them into TypeScript.

**Pull request merged**